dropwizard-core/src/main/java/io/dropwizard/server/SimpleServerFactory.java

```diff
@@ -136,4 +132,13 @@ public Server build(Environment environment) {
 
         return server;
     }
+
+    @Override
+    public void configure(Environment environment) {
```

Comment:
the context path logic is slightly different between the two. It would be nice to move them into `AbstractServerFactory`